Lib/asyncio/events.py

```diff
@@ -630,9 +630,8 @@ def __init__(self):
         self._local = self._Local()
 
     def get_event_loop(self):
-        """Get the event loop.
-
-        This may be None or an instance of EventLoop.
+        """Get the event loop for the current context.
+        Returns an instance of EventLoop or raises an exception.
```

Comment:
Need a newline before "Returns".  See https://www.python.org/dev/peps/pep-0257/ for the reference.